TeamA/AuthorityUser/src/main/java/org/wso2/sample/is/sso/agent/SampleContextEventListener.java

```
@@ -0,0 +1,83 @@
+
+    public void contextInitialized(ServletContextEvent servletContextEvent) {
+
+        properties = new Properties();
+        try {
+            if(servletContextEvent.getServletContext().getContextPath().contains("authorityuser")) {
+                properties.load(servletContextEvent.getServletContext().
+                        getResourceAsStream("/WEB-INF/classes/authorityuser.properties"));
+            } else if(servletContextEvent.getServletContext().getContextPath().contains("avis.com")) {
+                properties.load(servletContextEvent.getServletContext().
+                        getResourceAsStream("/WEB-INF/classes/avis.properties"));
+            }
+            InputStream keyStoreInputStream = servletContextEvent.getServletContext().
+                    getResourceAsStream("/WEB-INF/classes/wso2carbon.jks");
+            SSOAgentX509Credential credential =
+                    new SSOAgentX509KeyStoreCredential(keyStoreInputStream,
+                            properties.getProperty("KeyStorePassword").toCharArray(),
+                            properties.getProperty("IdPPublicCertAlias"),
+                            properties.getProperty("PrivateKeyAlias"),
+                            properties.getProperty("PrivateKeyPassword").toCharArray());
+            SSOAgentConfig config = new SSOAgentConfig();
+            config.initConfig(properties);
+            config.getSAML2().setSSOAgentX509Credential(credential);
+            config.getOpenId().setAttributesRequestor(new SampleAttributesRequestor());
+            servletContextEvent.getServletContext().
+                    setAttribute(SSOAgentConstants.CONFIG_BEAN_NAME, config);
+        } catch (IOException e){
+            LOGGER.log(Level.SEVERE, e.getMessage(), e);
+        } catch (SSOAgentException e) {
+            LOGGER.log(Level.SEVERE, e.getMessage(), e);
+        }
+    }
+
+    public void contextDestroyed(ServletContextEvent servletContextEvent) {
```

Shall we throw `UnsupportedOperationException` if this method is not implemented?

---

This is done through the following PR. 
https://github.com/wso2-incubator/qa-identity-solution/pull/21